test/CsrfMiddlewareTest.php

```diff
@@ -15,8 +15,8 @@
 
 class CsrfMiddlewareTest extends TestCase
 {
-    /** @var MockObject<CsrfGuardFactoryInterface> */
-    private $guardFactory;
+    /** @var MockObject&CsrfGuardFactoryInterface */
+    private MockObject $guardFactory;
```

Comment:
Shouldn't that be:
```php
/** @var MockObject&CsrfGuardFactoryInterface */
private CsrfGuardFactoryInterface $guardFactory;
```

Reply:
updated https://github.com/mezzio/mezzio-csrf/pull/15/commits/049102261d02070f5eed20985ce2e862da9aa811